### PR TITLE
i18n(zh-cn): Translated `tailwind.mdx`

### PR DESCRIPTION
**Pull request by vuecat**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

Translated `tailwind.mdx`.

### CODE REVIEW

File: src/content/docs/zh-cn/guides/integrations-guide/tailwind.mdx [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+**`astro.config.mjs`**
+
+```js
+import { defineConfig } from 'astro/config';
+import tailwind from '@astrojs/tailwind';
+
+export default defineConfig({
+  integrations: [tailwind({
+    // Example: Provide a custom path to a Tailwind config file
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/tailwind.mdx:102]:
```suggestion
    // 示例：为 Tailwind 配置文件提供一个自定义路径
```

File: src/content/docs/zh-cn/guides/integrations-guide/tailwind.mdx [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+要禁用这个默认行为，请将 `config.applyBaseStyles` 设置为 `false`。如果你需要定义你自己的 `base.css` 文件（例如包括一个[`@layer` 指令](https://tailwindcss.com/docs/functions-and-directives#layer)），这可能很有用。如果你不希望 `base.css` 被导入你的项目的每一个页面，这也很有用。
+
+**`astro.config.mjs`**
+
+```js
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+  integrations: [tailwind({
+    // Example: Disable injecting a basic `base.css` import on every page.
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/tailwind.mdx:128]:
```suggestion
    // 示例：在每个页面上禁用注入基本的`base.css`导入。
```

File: src/content/docs/zh-cn/guides/integrations-guide/tailwind.mdx [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+    // Useful if you need to define and/or import your own custom `base.css`.
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/tailwind.mdx:129]:
```suggestion
    // 如果你需要定义或导入你自己的 "base.css"，这很有用。
```

**Review by liruifengv — CHANGES_REQUESTED** [review]
@vuecat Thanks! Long wait. I translated some untranslated changes. Other LGTM!

**Review by Yan-Thomas — APPROVED** [review]
Looks great, thanks @vuecat & @liruifengv 🙌

**Pull request merged**